### PR TITLE
Update dependencies

### LINKED ISSUE #816: Sourcery fails when using `project` in configuration file with a project that uses swift packages.
Sourcery output:
```
error: while reading .yml 'YoufoodzTests/.sourcery.yml'. 'The element XCSwiftPackageProductDependency is not supported'
```

If the swift packages are removed from the project, Sourcery runs fine

Sourcery version: `17.0`

### PR DESCRIPTION
**Pull request by krzysztofzablocki**
Updates XcodeProj, Stencil and StencilSwiftKit amongst other things

Should fix #816 ?

### CODE REVIEW

**Comment by krzysztofzablocki** [comment]
Try running this version @vCrespoP @VasylKo and let me know if that helps with your issues?

**Review by ilyapuchka — APPROVED** [review]
looks good 👍

**Comment by RomanTysiachnik** [comment]
Awesome!

**Comment by vCrespoP** [comment]
> Try running this version @vCrespoP @VasylKo and let me know if that helps with your issues?

Yes it does. Thanks a lot for the effort!! 
:D

**Comment by vCrespoP** [comment]
> Try running this version @vCrespoP @VasylKo and let me know if that helps with your issues?

Ok sorry to come back to you, but I found that this version has a breaking change somehow. I was using this bit of code to check a condition:
```
variable.typeName.generic.name == "..."
```
But now that condition is not met anymore. I don't know what the breaking change could be but I tested on 1.0.2 and my code was generated fine and in 1.0.3. Same custom template, produces a different output because of that bit of code.

Any clues, hints or fixes?

Thanks again to everyone involved

**Comment by krzysztofzablocki** [comment]
can you be more specific in what the code that fails looks like ? print the variable typeName / type and show me the source file for that definition

**Comment by vCrespoP** [comment]
Yes, I'm providing more details to see if we can trace it:

We're using an Undefined enum to wrap variables in it. It's really specific for a super concrete requirement, no need to go into more detail here I assume:
```
enum Undefined<T> {
    case ...
}
```

Line of custom template:
`{% if variable.typeName.generic.name == "Undefined" %}`

When generating the code for a variable like this:
`var example: Undefined<Int>`

I've modified the template to print for variables the type and typeName like this:

Sourcery 1.0.2:
example: 
```
TYPENAME: Undefined<Int>
TYPE: Int? = nil
```

Sourcery 1.0.3:
example:
```
TYPENAME
                    Undefined<Int>
        TYPE
                    Enum: module = Optional("CarSharing"), imports = ["Foundation"], typealiases = [:], isExtension = false, kind = enum, accessLevel = internal, name = Undefined, isGeneric = true, localName = Undefined, variables = [Variable: name = value, typeName = Wrapped?, isComputed = true, isStatic = false, readAccess = internal, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = [:], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined), Variable: name = debugDescription, typeName = String, isComputed = true, isStatic = false, readAccess = public, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = ["public": public], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined)], methods = [Method: name = init(_ value: Wrapped), selectorName = init(_:), parameters = [MethodParameter: argumentLabel = nil, name = value, typeName = Wrapped, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = init(), selectorName = init, parameters = [], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = == (lhs: Undefined<Wrapped>, rhs: Undefined<Wrapped>), selectorName = ==(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = != <T>(lhs: Undefined<T>, rhs: Undefined<T>), selectorName = !=(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool where T: Equatable, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], subscripts = [], initializers = [Method: name = init(_ value: Wrapped), selectorName = init(_:), parameters = [MethodParameter: argumentLabel = nil, name = value, typeName = Wrapped, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = init(), selectorName = init, parameters = [], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], annotations = [:], staticVariables = [], staticMethods = [Method: name = == (lhs: Undefined<Wrapped>, rhs: Undefined<Wrapped>), selectorName = ==(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = != <T>(lhs: Undefined<T>, rhs: Undefined<T>), selectorName = !=(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool where T: Equatable, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], classMethods = [], instanceVariables = [Variable: name = value, typeName = Wrapped?, isComputed = true, isStatic = false, readAccess = internal, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = [:], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined), Variable: name = debugDescription, typeName = String, isComputed = true, isStatic = false, readAccess = public, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = ["public": public], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined)], instanceMethods = [Method: name = init(_ value: Wrapped), selectorName = init(_:), parameters = [MethodParameter: argumentLabel = nil, name = value, typeName = Wrapped, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = init(), selectorName = init, parameters = [], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], computedVariables = [Variable: name = value, typeName = Wrapped?, isComputed = true, isStatic = false, readAccess = internal, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = [:], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined), Variable: name = debugDescription, typeName = String, isComputed = true, isStatic = false, readAccess = public, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = ["public": public], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined)], storedVariables = [], inheritedTypes = ["CustomDebugStringConvertible", "Equatable", "Hashable", "Comparable"], containedTypes = [], parentName = nil, parentTypes = AnyIterator<Type>(_box: Swift._IteratorBox<Swift._ClosureBasedIterator<SourceryRuntime.Type>>), attributes = [:], cases = [EnumCase: name = undefined, rawValue = nil, associatedValues = [], annotations = [:], indirect = false, hasAssociatedValue = false, EnumCase: name = defined, rawValue = nil, associatedValues = [AssociatedValue: localName = nil, externalName = nil, typeName = Wrapped, defaultValue = nil, annotations = [:]], annotations = [:], indirect = false, hasAssociatedValue = true], rawTypeName = nil, hasAssociatedValues = true
                Undefined<Int>,
```

**Comment by vCrespoP** [comment]
> @vCrespoP can you be more specific in what the code that fails looks like ? print the variable typeName / type and show me the source file for that definition

Yes, I'm providing more details to see if we can trace it:

We're using an Undefined enum to wrap variables in it. It's really specific for a super concrete requirement, no need to go into more detail here I assume:

```
enum Undefined<T> {
    case ...
}
```

Line of custom template:

```
 {% if variable.typeName.generic.name == "Undefined" %}
```

When generating the code for a variable like this: 

```
var example: Undefined<Int>
```

I've modified the template to print the type and typeName for that variable like this:

```
{{ variable.typeName }}
{{ variable.type }}
```

Sourcery 1.0.2: example:
```
typeName: Undefined<Int>
type: (Nothing!)
GenericType: name = Undefined, typeParameters = [GenericTypeParameter: typeName = Int]
```

Sourcery 1.0.3: example:
```
TYPENAME: Undefined<Int>
TYPE: Enum: module = Optional("REMOVED"), imports = ["Foundation"], typealiases = [:], isExtension = false, kind = enum, accessLevel = internal, name = Undefined, isGeneric = true, localName = Undefined, variables = [Variable: name = value, typeName = Wrapped?, isComputed = true, isStatic = false, readAccess = internal, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = [:], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined), Variable: name = debugDescription, typeName = String, isComputed = true, isStatic = false, readAccess = public, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = ["public": public], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined)], methods = [Method: name = init(_ value: Wrapped), selectorName = init(_:), parameters = [MethodParameter: argumentLabel = nil, name = value, typeName = Wrapped, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = init(), selectorName = init, parameters = [], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = == (lhs: Undefined<Wrapped>, rhs: Undefined<Wrapped>), selectorName = ==(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = != <T>(lhs: Undefined<T>, rhs: Undefined<T>), selectorName = !=(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool where T: Equatable, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], subscripts = [], initializers = [Method: name = init(_ value: Wrapped), selectorName = init(_:), parameters = [MethodParameter: argumentLabel = nil, name = value, typeName = Wrapped, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = init(), selectorName = init, parameters = [], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], annotations = [:], staticVariables = [], staticMethods = [Method: name = == (lhs: Undefined<Wrapped>, rhs: Undefined<Wrapped>), selectorName = ==(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<Wrapped>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = != <T>(lhs: Undefined<T>, rhs: Undefined<T>), selectorName = !=(_:_:), parameters = [MethodParameter: argumentLabel = nil, name = lhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:], MethodParameter: argumentLabel = nil, name = rhs, typeName = Undefined<T>, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = Bool where T: Equatable, `throws` = false, `rethrows` = false, accessLevel = , isStatic = true, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], classMethods = [], instanceVariables = [Variable: name = value, typeName = Wrapped?, isComputed = true, isStatic = false, readAccess = internal, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = [:], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined), Variable: name = debugDescription, typeName = String, isComputed = true, isStatic = false, readAccess = public, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = ["public": public], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined)], instanceMethods = [Method: name = init(_ value: Wrapped), selectorName = init(_:), parameters = [MethodParameter: argumentLabel = nil, name = value, typeName = Wrapped, `inout` = false, typeAttributes = [:], defaultValue = nil, annotations = [:]], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:], Method: name = init(), selectorName = init, parameters = [], returnTypeName = CarSharing.Undefined, `throws` = false, `rethrows` = false, accessLevel = internal, isStatic = false, isClass = false, isFailableInitializer = false, annotations = [:], definedInTypeName = Optional(Undefined), attributes = [:]], computedVariables = [Variable: name = value, typeName = Wrapped?, isComputed = true, isStatic = false, readAccess = internal, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = [:], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined), Variable: name = debugDescription, typeName = String, isComputed = true, isStatic = false, readAccess = public, writeAccess = , isMutable = false, defaultValue = nil, annotations = [:], attributes = ["public": public], isFinal = false, isLazy = false, definedInTypeName = Optional(Undefined), actualDefinedInTypeName = Optional(CarSharing.Undefined)], storedVariables = [], inheritedTypes = ["CustomDebugStringConvertible", "Equatable", "Hashable", "Comparable"], containedTypes = [], parentName = nil, parentTypes = AnyIterator<Type>(_box: Swift._IteratorBox<Swift._ClosureBasedIterator<SourceryRuntime.Type>>), attributes = [:], cases = [EnumCase: name = undefined, rawValue = nil, associatedValues = [], annotations = [:], indirect = false, hasAssociatedValue = false, EnumCase: name = defined, rawValue = nil, associatedValues = [AssociatedValue: localName = nil, externalName = nil, typeName = Wrapped, defaultValue = nil, annotations = [:]], annotations = [:], indirect = false, hasAssociatedValue = true], rawTypeName = nil, hasAssociatedValues = true
                Undefined<Int>,

GenericType: name = REMOVED.Undefined, typeParameters = [GenericTypeParameter: typeName = Int]
* REMOVED == module
```

I guess I can go with 1.0.3 updating the template to get those values somehow differently (instead of Undefined, module.Undefined?), but I was wondering why this changed so much and if it was related to updating the dependencies.

Thanks again @krzysztofzablocki !

**Comment by krzysztofzablocki** [comment]
it wasn't just related to updating dependencies since we also fixed type resolution across modules in 1.0.3 that's why you need to specify module name, since that's fully qualified name

**Comment by vCrespoP** [comment]
Oh! Sorry for the confusion then @krzysztofzablocki !
I was not expecting a breaking change in a minor version update. 

Is this it then?
https://github.com/krzysztofzablocki/Sourcery/blob/master/CHANGELOG.md

Could you please suggest me how to define the following code in 1.0.3 so it behaves as expected?

1.0.2
```
 {% if variable.typeName.generic.name == "Undefined" %}
``` 

This needs to be changed too to strip the module in the generated code:
1.0.2
```
{{ variable.typeName.generic.typeParameters.first.typeName }}
```
As it's printing `CarSharing.String?` instead of `String?`

Thanks, this is my last doubt regarding this matter :)

**Comment by krzysztofzablocki** [comment]
It's not really API change but a bug fix for modules because when you use sources instead of target xcodeproj nothing has changed, before there was a bug that made your types unknown to the parser.

String is unknown type to Sourcery since it's Foundation type, but I'm guessing you are doing some extension in CarSharing on String which brings this type into our known types system ?

**Comment by vCrespoP** [comment]
I have some extensions for String, but the String definition is in Foundation, of course. I don't know why I get that module prefixed, you know way more than I do on this topic so I don't have an answer for that question :D

I'm quite confused then about what do I have to change to fit these changes into my current template. 

This could be a bugfix but for me it's a quite big breaking change:
- Conditionals do not behave in the same way because of the value to be compared
- It does not generate the same code it did
- It doesn't compile anymore as well.

Should I loop differently through the structures?
```
for type in types.implementing.AutoInitUndefined
```

Should I append somehow the module to the checks?
```
 {% if variable.typeName.generic.name == "{{ variable.module }}.Undefined" %} <~ is this even correct?
```

Should I strip the `module.` if present with my generated code so it looks like it was before?
```
CarSharing.String vs String
```

Also, if you believe this belongs to a different issue, let me know to follow the discussion there. 
Thanks once again.

**Comment by krzysztofzablocki** [comment]
the `Undefined` check should always use the module in which `Undefined` is defined since it's known type, so you just need to update the value you check for to be `module.Undefined` 

in regards to `CarSharing.String` the problem is that Sourcery doesn't know `String` exists, so when you extend `String` you make it aware of that type and since it doesn't have any more information about it it assumes this is type defined in `CarSharing` because that's the module in which it can see that extension.
I'll need to think about this behaviour, maybe we'll change it to not add module names in extensions of undefined types 🤔 

as for getting rid of the module name you should be able to use `replace` filter https://github.com/SwiftGen/StencilSwiftKit/blob/stable/Documentation/filters-strings.md#filter-replace

**Comment by vCrespoP** [comment]
> I'll need to think about this behaviour, maybe we'll change it to not add module names in extensions of undefined types 🤔 

I think that's a good idea!

I've managed to fix my templates, but now, they are completely coupled to my project, as I can't find another way of dealing with the problems I mentioned in earlier posts:
1) Specify the module:
```
{% if variable.typeName.generic.name == "Undefined" %} <~Before
{% if variable.typeName.generic.name == "CarSharing.Undefined" %} <~ After
```
I've tried without success:
- `module.Undefined`
- `"{{ variable.module }}.Undefined"`
- `{{ variable.module }}.Undefined`
- ...

2) Filter the module for some types, specially the String:
```
{{ variable.typeName.generic.typeParameters.first.typeName }}? = nil <~ Before
{{ variable.typeName.generic.typeParameters.first.typeName|replace:"CarSharing.","" }}? = nil <~ After
```

So basically, I just added the module explicitly everywhere, which I'd like to avoid. Also, and again, this bugfix contained breaking changes for every template that handles generics and I'm concerned I'm not the only one out there.

Appreciate the great communication here!

**Comment by krzysztofzablocki** [comment]
@vCrespoP I didn't mean to use `module` as actual name, I just didn't know if the `Undefined` was defined in `CarSharing` or somewhere else, you need to use the definition module, this has nothing to do with variable definitions as I said, `Undefined` fully qualified name isn't `Undefined`, it's `CarSharing.Undefined`.

This bugfix didn't have breaking changes for any templates using generics, it has little to do with generics, it has breaking changes when using pbxproj integration instead of our default `sources` but that's because it wasn't working correctly before so your types weren't attributed with module names as they should when using module support so that you can properly cross inter-module boundaries.

If you define same basic type name in 2 modules e.g. `Result`, if you use module integration then **you need to differentiate between `Module1.Result` and `Module2.Result`**, one of the PR's from community introduced bug that broke module resolution which is why your previous version didn't have it but that was side-effect of a bug and not how you should be writting your code when using modules.

And you can make your template work across different projects by extracting the module name into arguments, same way as I have arguments in https://github.com/krzysztofzablocki/AutomaticSettings

**Comment by vCrespoP** [comment]
> And you can make your template work across different projects by extracting the module name into arguments, same way as I have arguments in https://github.com/krzysztofzablocki/AutomaticSettings 

This helped me for sure! 

And also, many thanks for the explanation @krzysztofzablocki 💪🏻

**Comment by VasylKo** [comment]
@krzysztofzablocki thank's a lot for the update! All works fine in my case.
